google/cloud/bigtable/examples/bigtable_examples_common.cc

```diff
@@ -171,10 +172,7 @@ google::cloud::bigtable::examples::Commands::value_type MakeCommandEntry(
         argv.size() != 3 + args.size()) {
       std::ostringstream os;
       os << name << " <project-id> <instance-id> <table-id>";
-      char const* sep = " ";
-      for (auto const& a : args) {
-        os << sep << a;
-      }
+      os << absl::StrJoin(args, " ");
```

Comment:
These look like they drop the initial separator.

Reply:
Good catch, thanks.

Reply:
It looks like they are now different when `args` is empty.